agorauikit_android/src/main/java/io/agora/agorauikit_android/AgoraVideoViewer+IRtcEngineEventHandler.kt

```diff
@@ -123,8 +130,9 @@ open class AgoraVideoViewerHandler(private val hostView: AgoraVideoViewer) : IRt
         channel?.let {
             this.hostView.delegate?.joinedChannel(it)
         }
-        if (!this.hostView.connectionData.username.isNullOrEmpty()) {
-            this.hostView.createRtmChannel()
+        this.hostView.isInRtcChannel = true
+        if (!this.hostView.agoraRtmController.isLoggedIn) {
```

Comment:
Looks like this will still fire if rtm is not enabled. Can you confirm it is okay?